wiki/Wiki.md

```diff
@@ -0,0 +1,80 @@
+## Table of contents
+
+* [Contributing to the wiki](#contributing-to-the-wiki)
+   * [Opening a pull request](#opening-a-pull-request)
+   * [Failed to push changes to wiki upon PR merge.](#failed-to-push-changes-to-wiki-upon-pr-merge)
+* [Implementation details](#implementation-details)
+   * [Wiki architecture](#wiki-architecture)
+   * [When you make changes through the web interface](#when-you-make-changes-through-the-web-interface)
+   
+
+## Contributing to the wiki
+
+If you notice something about the wiki that could be improved, please let us know! There are a couple of ways you can do so:
+
+* If the improvement you have in mind is big, you want feedback before you start working on it, or you don't have time to make the change yourself, [open an issue](https://github.com/oppia/oppia-web-developer-docs/issues/new/choose) in the source repository.
```

Comment:
Replace the `open an issue` link from oppia-web to oppia-android

Reply:
Done